apps-devstg/us-east-1/k8s-eks/identities/ids_external_secrets.tf

```diff
@@ -51,7 +51,7 @@ resource "aws_iam_policy" "external_secrets_secrets_manager" {
         "kms:DescribeKey"
       ],
       "Resource": [
-        "${data.terraform_remote_state.shared.outputs.aws_kms_key_arn}"
+        "${data.terraform_remote_state.shared-keys.outputs.aws_kms_key_arn}"
```

Comment:
a bug I found while creating the identities of the cluster, the state is called `shared-keys`, not `shared`